lighthouse-core/test/audits/font-display-test.js

```diff
@@ -5,89 +5,110 @@
  */
 'use strict';
 
-const NetworkRequest = require('../../lib/network-request');
 const Audit = require('../../audits/font-display.js');
 const assert = require('assert');
 const Runner = require('../../runner.js');
 const networkRecordsToDevtoolsLog = require('../network-records-to-devtools-log.js');
 
 /* eslint-env jest */
-const openSansFont = {
-  display: 'auto',
-  family: 'open Sans',
-  stretch: 'normal',
-  style: 'normal',
-  weight: '400',
-  src: [
-    'https://fonts.gstatic.com/s/opensans/v15/u-WUoqrET9fUeobQW7jkRYX0hVgzZQUfRDuZrPvH3D8.ttf',
-    'https://fonts.gstatic.com/s/opensans/v15/u-WUoqrET9fUeobQW7jkRYX0hVgzZQUfRDuZrPvH3D8.woff2',
-  ],
-};
-const openSansFontBold = {
-  display: 'auto',
-  family: 'open Sans',
-  stretch: 'normal',
-  style: 'normal',
-  weight: '600',
-  src: [
-    'https://fonts.gstatic.com/s/opensans/v15/k3k702ZOKiLJc3WVjuplzA7aC6SjiAOpAWOKfJDfVRY.woff2',
-  ],
-};
 
 describe('Performance: Font Display audit', () => {
-  function getArtifacts(networkRecords, fonts) {
+  let networkRecords;
+  let stylesheet;
+
+  beforeEach(() => {
+    stylesheet = {content: ''};
+  });
+
+  function getArtifacts() {
     return Object.assign({
       devtoolsLogs: {[Audit.DEFAULT_PASS]: networkRecordsToDevtoolsLog(networkRecords)},
-      Fonts: fonts,
+      URL: {finalUrl: 'https://example.com/foo/bar/page'},
+      CSSUsage: {stylesheets: [stylesheet]},
     }, Runner.instantiateComputedArtifacts());
   }
 
-  it('fails when not all fonts have a correct font-display rule', () => {
-    const webFonts = [
-      Object.assign({}, openSansFont, {display: 'block'}),
-      openSansFontBold,
-    ];
+  it('fails when not all fonts have a correct font-display rule', async () => {
+    stylesheet.content = `
+      @font-face {
+        src: url("./font-a.woff");
```

Comment:
maybe call out the quote differences here? Either a comment (or comments for each in the css string) or even a test only testing quoting the url? (though it would be identical to this test :)

It's kind of not obvious that this is covering those possibilities if you haven't just read the audit, and I could see them being accidentally dropped in future edits to this file without knowing they were important.

Reply:
yeah good call, done